modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -1313,6 +1313,36 @@ TEST_F(GAPI_Streaming_Unit, SetSource_After_Completion)
     EXPECT_EQ(0., cv::norm(out, out_ref, cv::NORM_INF));
 }
 
+inline void checkPullOverload(const cv::Mat& ref,
```

Comment:
I believe it must be inside `anonymous` namespace instead of `inline`.

Reply:
Fixed.